lib/improver/blending/weighted_blend.py

```diff
@@ -870,51 +871,28 @@ def process(self, cube, weights=None):
         # Check to see if the data is percentile data
         perc_coord = self.check_percentile_coord(cube)
 
-        # Create slices over the threshold coordinate
-        try:
```

Comment:
I also found that this slicing over threshold was unnecessary and complicated things when you had 3D weights, so I removed it.

Reply:
Huzzah, simplification!

Reply:
:+1: